apps/web/src/config/constants/endpoints.ts

```diff
@@ -32,6 +32,8 @@ export const BLOCKS_CLIENT_LINEA = BLOCKS_SUBGRAPHS[ChainId.LINEA]
 export const BLOCKS_CLIENT_BASE = BLOCKS_SUBGRAPHS[ChainId.BASE]
 export const BLOCKS_CLIENT_OPBNB = BLOCKS_SUBGRAPHS[ChainId.OPBNB]
 export const STABLESWAP_SUBGRAPH_CLIENT = 'https://api.thegraph.com/subgraphs/name/pancakeswap/exchange-stableswap'
+export const STABLESWAP_SUBGRAPH_CLIENT_ARBITRUM =
```

Comment:
let's keep single source of truth. Use `STABLESWAP_SUBGRAPHS` from chains

Reply:
Done